app/services/EncryptionService.scala

```diff
@@ -120,7 +121,8 @@ class EncryptionService @Inject()(secureGCMCipher: SecureGCMCipher, appConfig: A
       paymentsIntoPension = decryptPaymentsIntoPensionViewModel(pension.encryptedPaymentsIntoPension),
       pensionsAnnualAllowances = decryptPensionAnnualAllowanceViewModel(pension.encryptedPensionAnnualAllowances),
       pensionLifetimeAllowances = pension.pensionLifetimeAllowances.decrypted(),
-      incomeFromPensions = pension.incomeFromPensions.decrypted()
+      incomeFromPensions = pension.incomeFromPensions.decrypted(),
+      unauthorisedPayments = pension.unauthorisedPayments.decrypted()
```

Comment:
Could this be moved into `PensionsCYAModel` now (both `decryptPensions` and `encryptPension`)? 

Or should we wait until [SASS-3049](https://jira.tools.tax.service.gov.uk/browse/SASS-3049)?

(I may have misunderstood)